packages/vite/src/node/cli.ts

```diff
@@ -263,21 +264,30 @@ cli
     `[boolean] force empty outDir when it's outside of root`,
   )
   .option('-w, --watch', `[boolean] rebuilds when modules have changed on disk`)
+  .option('--all', `[boolean] build all environments`)
```

Comment:
Yes, I think we should support `--environment`. It may only make sense in some projects though because some frameworks may require a strict orchestration. Added it here: https://github.com/vitejs/vite/pull/16129/commits/96e825a48d8da2f3ce6238b11d16c417b1696671